.circleci/config.yml

```diff
@@ -1,604 +1,604 @@
-              scale up services for integration tests,
-            ]
-          filters: { branches: { only: [main] } }
-          task_name: integration_test_v2
-          tf_workspace: integration
-          override: "sh,./tests/Behat/run-tests-parallel.sh,--tags,@v2_reporting_1"
-          timeout: 1200
-      - run-task:
-          name: integration tests v2 reports 2
-          requires:
-            [
-              integration tests reset DB,
-              scale up services for integration tests,
-            ]
-          filters: { branches: { only: [main] } }
-          task_name: integration_test_v2
-          tf_workspace: integration
-          override: "sh,./tests/Behat/run-tests-parallel.sh,--tags,@v2_reporting_2"
-          timeout: 1200
-      - run-task:
-          name: integration tests v2 admin
-          requires:
-            [
-              integration tests reset DB,
-              scale up services for integration tests,
-            ]
-          filters: { branches: { only: [main] } }
-          task_name: integration_test_v2
-          tf_workspace: integration
-          override: "sh,./tests/Behat/run-tests-parallel.sh,--tags,@v2_admin"
-          timeout: 1200
-      - run-task:
-          name: integration tests sequential only
-          requires:
-            [
-              integration tests reset DB,
-              scale up services for integration tests,
-            ]
-          filters: { branches: { only: [main] } }
-          task_name: integration_test_v2
-          tf_workspace: integration
-          override: "sh,./tests/Behat/run-tests.sh,--tags,@v2_sequential,--profile,v2-tests-browserkit"
-          timeout: 1200
-      - scale-services:
-          name: scale down services for integration tests
-          replicas: "1"
-          profile: digideps-ci-pre
-          tf_workspace: integration
-          requires:
-            [
-              integration tests v2 reports 1,
-              integration tests v2 reports 2,
-              integration tests v2 admin,
-              integration tests sequential only,
-            ]
-          filters: { branches: { only: [main] } }
-      - client-unit-test:
-          name: client unit test
-          requires: [apply integration]
-          filters: { branches: { only: [main] } }
-      - api-unit-tests:
-          name: api unit test
-          requires: [apply integration]
-          filters: { branches: { only: [main] } }
-      - cross-browser-test:
-          name: cross browser test
-          requires: [apply integration]
-          filters: { branches: { only: [main] } }
-
-  scheduled_workspace_deletion:
-    triggers:
-      - schedule:
-          cron: "0 8,20 * * 0-6" # twice a day at 8am and 8pm
-          filters: { branches: { only: [main] } }
-    jobs:
-      - destroy-workspaces:
-          name: destroy non protected workspaces
-          filters: { branches: { only: [main] } }
+#  dependabot_app:
```

Comment:
Any particular reason for not just deleting this and using git history if you need to check it?